### ISSUE TITLE
Button Panic

### ISSUE THREAD

**Issue by andrecordeiro** [tananaev/traccar-client-android#197]
Hy guys,

I would like to leave the Service Running Off, and when you press the POWER button for 3 consecutive times, even with SREEN_OFF , the service is running on. It would be a panic button. It's possible?

Best Regards.

**Comment by tananaev** [comment]
I don't think it's possible. Especially with locked screen.

**Comment by andrecordeiro** [comment]
this application does this (https://www.panicbutton.io/)  , but the traccar is much better in my opinion and only lacked this functionality to help people in a panic. tks my friend

**Comment by tananaev** [comment]
On the website it says:

> You should be able to use Panic Button from Android Gingerbread (2.3.3) to KitKat (4.4.2)

Which basically means that it works only on 21.5% of the devices. From the Google Play listing, it seems like there are a lot of device-specific issues as well, so the number is even lower.

**Comment by andrecordeiro** [comment]
Exactly, that's the problem , only works on Gingerbread ( 2.3.3 ) / KitKat ( 4.4.2 ) , so I figured to have this functionality in traccar . Thank you very much.

**Comment by tananaev** [comment]
I guess it works only on old versions for a reason.
